### PR TITLE
UI tweaks (Splash screen & QR code) to first-timer run of sample App

### PR DESCRIPTION
**Pull request by jumaallan**
Story: https://app.shortcut.com/smileid/story/10931/

## Summary

UI tweaks (Splash screen & QR code) to first-timer run of sample App

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

### CODE REVIEW

**Comment by jumaallan** [comment]
Samuel is working on a lottie file to animate the Smile Logo

File: sample/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -42,9 +42,18 @@
     <string name="about_us_visit_our_website">Visit our website</string>
     <string name="about_us_contact_support">Contact support</string>
 
+    <!--  Root Screen  -->
+    <string name="root_welcome">Welcome to our Sample App!</string>
+    <string name="root_description">To begin testing on this app, you need to add a configuration from Smile Portal. \n\nSee link here: \n<annotation is_url="true">portal.smileidentity.com/sdk</annotation></string>
+    <string name="root_scan_qr">Scan Configuration QR</string>
+    <string name="root_add_config">Add Config manually</string>
+    <string name="root_config_added">Configuration Added</string>
+    <string name="root_config_added_welcome">Welcome partner_%1$s, you can now proceed to the home screen of the Sample App</string>
```
Comment (vanshg) [inline sample/src/main/res/values/strings.xml:51]:
```suggestion
    <string name="root_config_added_welcome">Welcome Partner %1$s, you can now proceed to the home screen of the Sample App</string>
```

File: sample/src/main/java/com/smileidentity/sample/compose/components/SmileConfigModalBottomSheet.kt [hunk 2]
```diff
@@ -102,23 +88,6 @@ fun SmileConfigModalBottomSheet(
             ) {
                 Text(stringResource(id = R.string.settings_update_smile_config))
             }
-            OutlinedButton(
```
Comment (vanshg) [inline sample/src/main/java/com/smileidentity/sample/compose/components/SmileConfigModalBottomSheet.kt:105]: Why are we removing the ability to scan QR code from settings?

File: sample/src/main/java/com/smileidentity/sample/compose/RootScreen.kt [hunk 3]
```diff
@@ -32,24 +58,136 @@ import timber.log.Timber
...
     val client = remember {
         SmileID.getOkHttpClientBuilder()
             .addInterceptor(ChuckerInterceptor.Builder(context).build())
             .build()
     }
-    SmileIDTheme {
-        Surface(modifier = modifier) {
-            if (runtimeConfig != null) {
+
+    Scaffold(
```
Comment (vanshg) [inline sample/src/main/java/com/smileidentity/sample/compose/RootScreen.kt:97]: Why are we adding a Scaffold here? MainScreen already has a Scaffold

File: sample/src/main/java/com/smileidentity/sample/compose/RootScreen.kt [hunk 4]
```diff
@@ -32,24 +58,136 @@ import timber.log.Timber
...
+        modifier = modifier
+            .fillMaxSize(),
+    ) { paddingValues ->
+        BottomPinnedColumn(
```
Comment (vanshg) [inline sample/src/main/java/com/smileidentity/sample/compose/RootScreen.kt:101]: Bump on my previous comment -- can you pull out the smile config entry into its own composable, outside of the RootScreen?

**Review by vanshg — CHANGES_REQUESTED** [review]
When I launch the app, I always see the config entry screen momentarily, even though I've already defined the config. This shouldn't happen

File: sample/src/main/java/com/smileidentity/sample/compose/components/SmileConfigModalBottomSheet.kt (hunk 2 above)
Comment (jumaallan) [reply sample/src/main/java/com/smileidentity/sample/compose/components/SmileConfigModalBottomSheet.kt:105]: Has been removed on the new design - so it is an option on the main screen

**Review by vanshg — APPROVED** [review]
